README.md

```diff
@@ -1,14 +1,55 @@
 ![NFT Gallery Logo](/content/images/logo.png)
+
 # NFT Gallery
-NFT Gallery is a static site generator (SSG) that builds a gallery of your collected and created NFTs to showcase on your website. 
-
-## How it works:
-1. First, using Wallet Connect you will connect with your Ethereum wallet of choice
-2. NFT Gallery will pull in all of the NFTs that you currently collect or have created
-3. Next you can hide specific NFTs that you do not want to include in your gallery
-4. Pick a theme for your gallery
-5. Download a ZIP file with your static site, or deploy it into a Git repository of your choice
-6. Enjoy your new public gallery that's all your own!
-
-## How to deploy:
-To come
+
+NFT Gallery is a static site generator (SSG) that builds a gallery of your collected and created NFTs to showcase on your website.
+
+# Quickstart
+
+If you want to run your own instance of rodeo, generally the best way is to use our CLI tool
+
+```
+npx giddyup mygallery
+```
+
+After a few questions this will create a new rodeo project in the ./mygallery directory.
```

Comment:
Maybe some more stuff about authenticating a wallet?

Reply:
Ya need to figure out what that looks like first.